CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * Use reachability API to minimize the reconnection delay if the network
   connection was lost.
+* Add a new API for changing and retrieving permissions. This API can be used
```

Comment:
Do we want these APIs to replace the existing ones? I don't feel comfortable making a unilateral decision about whether we should commit to deprecation.